robocop/checkers/comments.py

```diff
@@ -38,17 +49,40 @@
         """,
     ),
     "0702": Rule(
+        RuleParam(
+            name="block",
+            default="^###",
+            converter=regex,
+            desc="Define regex for block comment.",
```

Comment:
Rather than using verb, we should describe the parameter, ie: ``Block comment regex pattern``

Reply:
fixed,